### DISCUSSION TITLE
Payment form errors don't work

### DISCUSSION THREAD

**Discussion by umkasanki** [solspace/craft-freeform#181] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
Ajax form errors only work correctly if no payment field is added to the page.

Correct work:
https://user-images.githubusercontent.com/496132/127286500-b2b6a120-34b7-461a-a3ca-41a17063ff09.mp4

Not correct work:
https://user-images.githubusercontent.com/496132/127286605-b254be3d-8b6f-4812-adff-f5607358e142.mp4

**Steps to reproduce**

1. Add form
2. Try to submit with empty fields

**Expected behavior**
Show field errors like it works without payment field

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Pro 3.7.5
- Freeform Version:  3.11.10
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh

**Additional context**

**Comment by gustavs-gutmanis** [comment, marked as answer]
The form validation happens only after Stripe has successfully validated your payment details.

**Comment by kjmartens** [comment]
As @gustavs-gutmanis mentions, this is currently a limitation right now, as Stripe validation takes precedence in the validation order. We'd like to improve this in the future though, if at all possible. :)

**Reply by umkasanki** [reply]
This is very strange anyway. No error messages appear under any field on the payment page. This greatly impairs the usability of the payment form. But this is the most important form on the site...

**Reply by kjmartens** [reply]
Errors will appear on other fields once the Stripe credit card fields are filled in. It just can't happen all at once, unfortunately.

**Reply by umkasanki** [reply]
Unfortunately, the client will not be satisfied with this answer (
